Prefetching-Library/android_prefetching_lib/src/androidTest/java/nl/vu/cs/s2group/nappa/room/dao/ActivityVisitTimeDaoTest.java

```diff
@@ -0,0 +1,89 @@
+package nl.vu.cs.s2group.nappa.room.dao;
+
+import android.content.Context;
+
+import androidx.room.Room;
+import androidx.test.core.app.ApplicationProvider;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import nl.vu.cs.s2group.nappa.room.ActivityData;
+import nl.vu.cs.s2group.nappa.room.PrefetchingDatabase;
+import nl.vu.cs.s2group.nappa.room.data.ActivityVisitTime;
+import nl.vu.cs.s2group.nappa.room.data.Session;
+import nl.vu.cs.s2group.nappa.util.TestUtil;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class ActivityVisitTimeDaoTest {
```

Comment:
Not particualrly necessary for this feature, but I did a quick experimentation with the test in Android.